### DISCUSSION TITLE
Meaning of: Skipping <PlexGuid:local://xxxx> because provider 'local' has no external Id ?

### DISCUSSION THREAD

**Discussion by MusicMunkee** [Taxel/PlexTraktSync#1845] [Q&A] [answered]
Hi,
just installed plextraktsync new on a new virtual machine. Now i get a decent amout of Warnings like this: Warning: Skipping <PlexGuid:local://xxxx> because provider 'local' has no external Id.

What does this warning exactly means? And how can it be repaired maybe?
What is meant by provider "local"?

It sounds like PlexTraktSync  is skiping these movies or tv shows, and thats a thing I dont want (nobody wants that probably :), so some help for preventing this would be great!

Here are original warnings:
`
WARNING  Antlers (2021): Skipping <PlexGuid:local://54847> because provider 'local' has no external Id                                                                
WARNING  Bigbug (2022): Skipping <PlexGuid:local://54850> because provider 'local' has no external Id
`

My PlexTraktSync is newest Version 0.29.4 running on debian LXC container on proxmox.

**Comment by simonc56** [comment, marked as answer]
`local` guid mean the media is unmatched in Plex. 
You need to [fix Match](https://support.plex.tv/articles/201018497-fix-match-match/).
